### PR TITLE
Added installation section and short administration guide

### PR DESCRIPTION
**Pull request by Monstrofil**
Added installation section and short administration guide into current docs.

### CODE REVIEW

**Comment by Monstrofil** [comment]
Also I removed a bunch of unused images from repository.

File: docs/installation/README.md [hunk 1]
```diff
@@ -1,309 +1,245 @@
...
-### Hardware compatibility
 
-CloudLinux supports all the hardware supported by RHEL/CentOS, with few exceptions. Exceptions are usually hardware that require binary drivers, and that doesn't have any open source alternatives.
-
-:::tip Note
-CloudLinux OS does not support ARM-based CPUs (e.g. Graviton)
-:::
+## Hardware compatibility
 
+CloudLinux supports all the hardware supported by RHEL/CentOS, with few exceptions.
```
Comment (alutsiuk123) [inline docs/installation/README.md]: CloudLinux OS Solo

File: docs/installation/README.md [hunk 2]
```diff
@@ -1,309 +1,245 @@
...
+* Installs CloudLinux repository settings and imports CloudLinux RPM key.
+* Replaces redhat/centos-release, redhat-release-notes, redhat-logos with CL version.
+* Re-installs CL version of rhnlib/rhnplugin.
+* Checks for binary kernel modules, finds replacement if needed.
+* Detects OVH servers and fixes mkinitrd issues.
+* Detects Linode servers and fixes grub issues.
+* Checks if LES is installed.
+* Checks that <span class="notranslate">`/etc/fstab`</span> has correct <span class="notranslate">`/dev/root`</span>
+* Checks for efi.
+* Installs CloudLinux Manager for cPanel Solo.
```
Comment (alutsiuk123) [inline docs/installation/README.md:244]: why only cPanel Solo? I thought we support regular cPanel as well

File: docs/introduction/README.md [hunk 3]
```diff
@@ -15,7 +15,11 @@ We believe that this distribution will be useful for you if:
    CloudLinux OS Solo provides a set of tools for website monitoring and performance tracking out-of-box;
 2. as a service provider you sell VPS servers and some of your customers require OS with extensive features 
    for better security, support, and performance.
-3. as a customer you want to be sure that [product has 24/7 technical support](https://cloudlinux.zendesk.com/).
+
+In any case we provide you [professional support 24/7](https://cloudlinux.zendesk.com/) 
+to resolve all issues you may have with distro.
+
+3. as a customer you want to be sure that [product has .
```
Comment (alutsiuk123) [inline docs/introduction/README.md]: [ - typo

File: docs/installation/README.md (hunk 2 above)
Comment (Monstrofil) [reply docs/installation/README.md:244]: We don't support resellers and block UI when more that a single user exists on server, so regular cPanel support is rather 'bug', not 'feature'.

File: docs/installation/README.md (hunk 1 above)
Comment (Monstrofil) [reply docs/installation/README.md]: done

File: docs/introduction/README.md (hunk 3 above)
Comment (Monstrofil) [reply docs/introduction/README.md]: Done

**Comment by Monstrofil** [comment]
@iatmachian please merge it.

**Pull request merged**